spec.emu

```diff
@@ -1087,8 +1130,16 @@ contributors: Google, Ecma International
           1. Let _ordinalMonth_ be _fields_.[[Month]].
         1. Let _day_ be _fields_.[[Day]].
         1. Assert: _day_ is not ~unset~.
-        1. If the date described by _calendar_, _arithmeticYear_, _ordinalMonth_, and _day_ does not exist, set _day_ to the closest day in the same month. If there are two equally-close dates in the same month, pick the later one.
-        1. Return an implementation-defined ISO Date Record that corresponds to the date described by _calendar_, _arithmeticYear_, _ordinalMonth_, and _day_.
+        1. Let _daysInMonth_ be CalendarDaysInMonth(_calendar_, _arithmeticYear_, _ordinalMonth_).
+        1. If _daysInMonth_ &lt; _day_, then
+          1. If _overflow_ is ~reject~, throw a *RangeError* exception.
```

Comment:
I realized that this RangeError isn't in the prose being edited, but I think it was wrongfully removed in #73. Previously the spec said

```
        1. If _fields_ describes an existing date in _calendar_, return an implementation-defined ISO Date Record that corresponds to the date described by _fields_.
        1. If _overflow_ is ~reject~, throw a *RangeError* exception.
```

So it would have thrown on an invalid day.